sktime/forecasting/arch/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Time series forecasting with arch models."""
+
+__all__ = ["ARCH", "StatsForecastARCH", "StatsForecastGARCH"]
+__author__ = ["eyjo", "Vasudeva-bit"]
+
+from sktime.forecasting.arch._stats_arch import StatsForecastARCH, StatsForecastGARCH
```

Comment:
It'll be better in my opinion if this module is named _statsforecast_arch instead of _stats_arch.

Reply:
Yeah, will change it.